torch_em/self_training/mean_teacher.py

```diff
@@ -235,6 +235,7 @@ def _train_epoch_unsupervised(self, progress, forward_context, backprop):
                 )
                 lr = [pm["lr"] for pm in self.optimizer.param_groups][0]
                 self.logger.log_lr(self._iteration, lr)
+                self.logger.log_ct(self._iteration, self.pseudo_labeler.confidence_threshold)
```

Comment:
Maybe we need to check here that the confidence_threshold is not None before logging?

Reply:
Yes, I did exactly that. Now it only logs the threshold if it is not None.